src/content/en/android/custom-tabs/headers/index.md

```diff
@@ -0,0 +1,178 @@
+
+
+<a name="table-1"></a>
+
+**Chrome version** | **CORS headers allowed**
+---------------- | ----------------------
+before v83       | whitelisted, non-whitelisted
+v83 to v85       | whitelisted
+v86 onwards      | whitelisted, non-whitelisted when a digital asset link is set up
+
+**Table 1.:** Filtering of non-whitelisted CORS headers.
+
+
+This guide shows how to set up a verified connection between the server and client and use that to send non-whitelisted http headers. You can skip to [Adding Extra Headers to CustomTab Intents for the code](#adding-extra-headers). 
+
+## Backgroud
+
+### Whitelisted vs. Non-whitelisted CORS Request Headers
+[Cross-Origin Resource Sharing (CORS)][6] allows a web application from one origin to request resources of a different origin. The list of **CORS-whitelisted** headers is maintained in the [HTML Standard][]. Example whitelisted headers are shown in [Table 2](#table-2). 
+
+<a name="table-2"></a>
+
+**Header**       | **Description**
+---------------- | ----------------------
+accept-language  | advertises natural languages the client understands
+content-language | describes language intended for the current audience
+content-type     | indicates the media type of the resource
+
+**Table 2.:** Example whitelisted CORS headers.
+
+The whitelisted headers are considered safe because they don’t contain sensitive user information and are unlikely to cause the server to perform potentially damaging operations.
+
+Examples of non-whitelisted headers are shown in [Table 3](#table-3).
+
+<a name="table-3"></a>
+
+**Header**   | **Description**
+-------------| ----------------------
+bearer-token | authenticates client at a server
+origin       | indicates origin of request
+cookie       | contains cookies set by server
+
+**Table 3.:** Example non-whitelisted CORS headers.
+
+Attaching non-whitelisted headers to CORS requests is discouraged by the HTML standard and servers assume that cross-origin requests contain only whitelisted headers. Sending non-whitelisted headers from cross-origin domains would allow malicious third-party apps to craft headers that misuse user cookies that Chrome (or another browser) stores and attaches to requests. The cookies could authenticate malicious server transactions that would otherwise not be possible.
+
+### Attaching Headers to Custom Tabs CORS Requests
```

Comment:
I think the request themselves are not CORS. The point here seems to be how to attach CORS white listed headers. WDYT?

```suggestion
### Attaching CORS whitelisted headers to Custom Tabs requests
```